local-config/collector.config.yaml

```diff
@@ -33,5 +33,5 @@ service:
   pipelines:
     traces/1:
       receivers: [otlp]
-      processors: [tail_sampling, batch]
```

Comment:
I removed this because its not in use anymore, we need to work on a fix if we want to add it back